host-bmc/host_pdr_handler.cpp

```diff
@@ -552,8 +572,29 @@ void HostPDRHandler::sendPDRRepositoryChgEvent(std::vector<uint8_t>&& pdrTypes,
                                                   nullptr, nullptr);
             if (record && pldm_pdr_record_is_remote(record))
             {
-                changeEntries[0].push_back(
-                    pldm_pdr_get_record_handle(repo, record));
+                if (!isHostUp())
+                {
+                    for (const auto& [terminusHandle, terminusInfo] : tlPDRInfo)
+                    {
+                        if (std::get<0>(terminusInfo) == terminusID &&
+                            std::get<1>(terminusInfo) == mctp_eid &&
+                            std::get<2>(terminusInfo))
+                        {
+                            if (record->terminus_handle == terminusHandle)
```

Comment:
this can be merged into the `if` in line number 579 with an &&. that could reduce one {} and makes it better in terms of readabilty.

Reply:
done